sdk/cosmosdb/cosmos/src/documents/DatabaseAccount.ts

```diff
@@ -97,5 +97,9 @@ export class DatabaseAccount {
 export interface Location {
   name: string;
   databaseAccountEndpoint: string;
-  unavailable?: boolean;
+  unavailable?: Unavailable;
+}
+
+export interface Unavailable {
```

Comment:
can we rename this interface to be more meaningful?

Reply:
Something signifying that is has to do with refresh of endpoints.